shared/elements/gaia_sim_picker/script.js

```diff
@@ -51,6 +53,21 @@ window.GaiaSimPicker = (function(win) {
     }
   };
 
+  /**
+   * Localize the component manually as l10n attributes are not supported
```

Comment:
I think we have a bug number for localization and components. I'll try to find it and put it in the comment.